src/User.js

```diff
@@ -94,10 +94,14 @@ export default class User {
   }
 
   get lastSentimentTimestamp() {
-    return this.userData.lastSentimentTimestamp;
+    return this.userData.lastSentiment && this.userData.lastSentiment.timestamp;
+  }
+
+  get lastSentimentLabel() {
+    return this.userData.lastSentiment && this.userData.lastSentiment.label;
```

Comment:
**Note to self:** run a quick migration to move over old fields into new object structure after deployment.